### PR TITLE
tests: visual regression for checking logo asset [WD-13063]

### PR DESCRIPTION
**Pull request by mas-who**
## Done
 - visual regression test for logo asset

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "start": "./entrypoint 0.0.0.0:${PORT}",
     "backend-run": "cd .. && make && go run ./cmd/lxd-site-mgrd --state-dir ./state/dir1/",
     "backend-init": "cd .. &&  go run ./cmd/lxd-site-mgr --state-dir ./state/dir1/ waitready && go run ./cmd/lxd-site-mgr --state-dir ./state/dir1/ init \"member1\" 0.0.0.0:9001 --bootstrap && scripts/generate_sites.sh --local && scripts/generate_configs.sh --local",
-    "test-e2e": "npx playwright test"
+    "test-e2e": "npx playwright test",
+    "test-e2e-screenshots": "npx playwright test --update-snapshots"
```
Comment (edlerd) [inline ui/package.json]:
```suggestion
    "test-e2e-update": "npx playwright test --update-snapshots"
```
Maybe clearer?

File: ui/tests/screenshot/logo.spec.ts-snapshots/logo-screenshot-1-chromium-linux.png
Comment (edlerd) [inline ui/tests/screenshot/logo.spec.ts-snapshots/logo-screenshot-1-chromium-linux.png:1]: Can we change this to have the same assets for chromium and firefox?

**Review by edlerd — CHANGES_REQUESTED** [review]

File: ui/tests/screenshot/logo.spec.ts-snapshots/logo-screenshot-1-chromium-linux.png
Comment (mas-who) [reply ui/tests/screenshot/logo.spec.ts-snapshots/logo-screenshot-1-chromium-linux.png:1]: done :+1:

**Review by edlerd — APPROVED** [review]
Code LGTM, test needs changes to become green though.

**Comment by mas-who** [comment]
@edlerd so looking at the CI failure, the chromium logo test is failing with a 7% pixel difference when compared to the screenshot. This is probably due to the screenshot being taken in firefox. Looking at the screenshots comparison, it almost feels like there is some kind of colour difference between firefox and chrome? Not sure if using the same screenshot for cross platform projects may result in these small differences all the time. We could try fix the pixel difference somehow, set a threshold for pixel difference in testing or go back to per project screenshots. wdyt? 

900b5bcc611bb6742af2ea338d68c11f2800f537
62e3a0a6dd73ce2473cd1f18083874bef96aced2
176ac2a1ffc9e9c733b4c54dcc629face4dd9a13

**Comment by edlerd** [comment]
> @edlerd so looking at the CI failure, the chromium logo test is failing with a 7% pixel difference when compared to the screenshot. This is probably due to the screenshot being taken in firefox. Looking at the screenshots comparison, it almost feels like there is some kind of colour difference between firefox and chrome? Not sure if using the same screenshot for cross platform projects may result in these small differences all the time. We could try fix the pixel difference somehow, set a threshold for pixel difference in testing or go back to per project screenshots. wdyt?

I see, thanks for figuring this out. Maybe easier in that case to revert to the previous case of having dedicated screenshots for each browser?

**Review by edlerd — APPROVED** [review]
LGTM, thanks for adding and figuring out the snapshot diff.

**Pull request merged**